README-zh_CN.md

```diff
@@ -34,9 +34,9 @@ Spiderpool 社区将最新的三个 Spiderpool 补丁版本视为稳定版本，
 
 |                         发布分支                                      |  更新时间    |                         镜像                                      |                               发行说明                                              |
 | -------------------------------------------------------------------- | ----------- | ---------------------------------------------------------------- | --------------------------------------------------------------------------------- |
-| [release-v0.9](https://github.com/spidernet-io/spiderpool/tree/release-v0.9) | 2023-12-28  | ``ghcr.io/spidernet-io/spiderpool/spiderpool-controller:v0.9.0`` | [Release Notes](https://github.com/spidernet-io/spiderpool/releases/tag/v0.9.0)   |
-| [release-v0.8](https://github.com/spidernet-io/spiderpool/tree/release-v0.8) | 2023-12-28  | ``ghcr.io/spidernet-io/spiderpool/spiderpool-controller:v0.8.4`` | [Release Notes](https://github.com/spidernet-io/spiderpool/releases/tag/v0.8.4)   |
-| [release-v0.7](https://github.com/spidernet-io/spiderpool/tree/release-v0.7) | 2023-12-28  | ``ghcr.io/spidernet-io/spiderpool/spiderpool-controller:v0.7.2`` | [Release Notes](https://github.com/spidernet-io/spiderpool/releases/tag/v0.7.2)   |
+| [release-v0.9](https://github.com/spidernet-io/spiderpool/tree/release-v0.9) | 2024-01-29  | ``ghcr.io/spidernet-io/spiderpool/spiderpool-controller:v0.9.1`` | [Release Notes](https://github.com/spidernet-io/spiderpool/releases/tag/v0.9.1)   |
```

Comment:
what a burden to maintain the version number, I figure we could remove the image and release time, and just tell the stable version information about 0.9  0.8 0.7

Reply:
ok，agree